### ISSUE TITLE
wasm port

### ISSUE THREAD

**Issue by BonsaiDen** [Gekkio/mooneye-gb#41]
Hey there,

first of all thanks for all the hard work. As an GB homebrew developer accurate emulation lies very close to my heart :)

Now since my evening was rather boring, I decided to check if it was possible to do a wasm port of mooneye-gb and low and behold it turned out to be rather easy with all the rust related tooling we have at hands these days: https://gitlab.com/BonsaiDen/mooneye-wasm

Hope you like it and keep on the good work!

**Comment by Gekkio** [comment]
Cool! Looks like my recent work on splitting core into a separate crate was useful here :smile:

**Comment by BonsaiDen** [comment]
Definitely! I was surprised myself how quick and easy it was to get it to run once the frame buffer was exposed to JS land :)

Now sound will be a different beast once support lands in the emulator, synchronization might end up getting a bit more tricky :see_no_evil:
